Project-Hotpot/Project-Hotpot/Algorithm/SongManager.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  SongManager.swift
+//  Project-Hotpot
+//
+//  Created by Harini Sundaram on 7/21/22.
+//
+
+import Foundation
+import Parse
+
+class SongManager : NSObject {
```

Comment:
mmm maybe add to managers folder? Great job adding more business logic layering in code <3